includes/widgets/icon-list.php

```diff
@@ -602,7 +604,7 @@ protected function render() {
 							<?php } ?>
 						</span>
 					<?php endif; ?>
-					<span <?php $this->print_render_attribute_string( $repeater_setting_key ); ?>><?php echo $item['text']; ?></span>
+					<span <?php $this->print_render_attribute_string( $repeater_setting_key ); ?>><?php echo esc_html( $item['text'] ); ?></span>
```

Comment:
Replace the `esc_html` with `$this->print_unescaped_setting` with the repeater args

Reply:
Done.